rust/src/serialization.rs

```diff
@@ -94,6 +95,17 @@ impl DeserializeEmbeddedGroup for Transaction {
         let witness_set = (|| -> Result<_, DeserializeError> {
             Ok(TransactionWitnessSet::deserialize(raw)?)
         })().map_err(|e| e.annotate("witness_set"))?;
+        let is_valid = (|| -> Result<_, DeserializeError> {
```

Comment:
We want to be permissive here in parsing. Be careful how you do it though since if it's `CBORType::Special` it could be *either* case (`null` in the `auxiliary_data / null`) or the bool `is_valid`.